redisCacheModule.js

```diff
@@ -36,7 +36,9 @@ function redisCacheModule(config){
   self.backgroundRefreshIntervalCheck = (typeof config.backgroundRefreshIntervalCheck === 'boolean') ? config.backgroundRefreshIntervalCheck : true;
   self.backgroundRefreshInterval = config.backgroundRefreshInterval || 60000;
   self.backgroundRefreshMinTtl = config.backgroundRefreshMinTtl || 70000;
+  self.JSON = config.JSON || Object.create(JSON);
```

Comment:
Could this just be `self.JSON = config.JSON || JSON;`?

Reply:
It can but it’s a bit error prone from what I experienced while writing the respective test case.

That’s because the `JSON` object is a global object and having redisCacheModule.JSON referencing it can lead to some inadvertent problems, e.g.: infinite recursion if the [overrode method calls into the global method](https://github.com/jpodwys/cache-service-redis/pull/13/files#diff-e817044058e7c3d8729f8a5724afe7bfR162).